src/hidra/receiver/plugins/asapo_producer.py

```diff
@@ -94,11 +89,14 @@ def get_exposed_path(metadata):
     return exposed_path
 
 
-def get_entry(dict_obj, name):
+def get_entry(dict_obj, name, default=None):
```

Comment:
This is tricky. What if the user wants to have `None` as default (sounds rather common to me).

Reply:
Changed.